notebooks/tutorial.raster_zonal_stats.ipynb

```diff
@@ -114,7 +114,7 @@
   },
```

Comment:
<p>shouldn't this be a call to <code>rzs.create_exactextract_zonal_stats</code> ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/geowrangler/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>